app/src/test/java/uk/ac/ebi/atlas/search/species/SpeciesSearchDaoIT.java

```diff
@@ -0,0 +1,99 @@
+package uk.ac.ebi.atlas.search.species;
+
+import com.google.common.collect.ImmutableSet;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+import uk.ac.ebi.atlas.solr.cloud.SolrCloudCollectionProxyFactory;
+
+import javax.inject.Inject;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.ac.ebi.atlas.solr.bioentities.BioentityPropertyName.SYMBOL;
+
+@WebAppConfiguration
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = TestConfig.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class SpeciesSearchDaoIT {
+    @Inject
+    private SolrCloudCollectionProxyFactory collectionProxyFactory;
+
+    private SpeciesSearchDao subject;
+
+    @BeforeEach
+    void setUp() {
+        subject = new SpeciesSearchDao(collectionProxyFactory);
+    }
+
+    @Test
+    void whenEmptySearchTextProvidedReturnEmptyOptional() {
```

Comment:
As I changed the code in the class under test now the test method name makes sense.

Reply:
Sorry, my bad. I misunderstood the test. You can make it more fluent with:
```java
assertThat(species).isEmpty();
```

Or if you want to make explicit that it’s checking the `Optional` rather than the `ImmutableSet`:
```java
assertThat(spcies).isNotPresnet();
```

You can have a look at all the `Optional` assertions here: https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractOptionalAssert.html.